packages/react-component-playground/src/components/welcome-screen.less

```diff
@@ -0,0 +1,20 @@
+.welcome-screen {
+  position: absolute;
+  left: 250px;
+  min-height: 100vh;
+  width: ~"calc(100% - 250px)";
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+
+  &-inner {
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif;
```

Comment:
Interesting choice of fonts. Maybe stay [consistent](https://github.com/react-cosmos/react-cosmos/blob/da85709af02f4ddf5b8d9908c52678f802a9f214/packages/react-component-playground/src/components/component-playground.less#L2) for now and later we can use some pretty web font.